chart/README.md

```diff
@@ -687,6 +687,12 @@ To upgrade Sonatype IQ Server and ensure a successful data migration, the follow
 | `iq_server.pvOwnershipOverrideResources.resources.requests.memory` | Persistence ownership initContainer request for memory resources in bytes                            | `nil`                      |
 | `iq_server.pvOwnershipOverrideResources.resources.limits.cpu`      | Persistence ownership initContainer limit for CPU resources in CPU units                             | `nil`                      |
 | `iq_server.pvOwnershipOverrideResources.resources.limits.memory`   | Persistence ownership initContainer limit for memory resources in bytes                              | `nil`                      |
+| `iq_server_jobs.migrationJobAnnotations`                           | Sonatype IQ DB Migration job Annotations                                                             | `nil`                      |
+| `iq_server_jobs.env`                                               | Sonatype IQ DB Migration job environment variables                                                   | `nil`                      |
```

Comment:
I see you added these iq_server_jobs variables but are they ever used in the iq-server-jobs template? I only see a reference for migrationJobAnnotations

Reply:
yes they are used in the template here: https://github.com/sonatype/nexus-iq-server-ha/blob/731051e2008f9e3a653229e4875933053b638abc/chart/templates/iq-server-jobs.yaml#L36